docs/supported_media.md

```diff
@@ -0,0 +1,311 @@
+fingerprinting such as [Privacy
+Budget](https://github.com/bslassey/privacy-budget) and [Brave's
+protections](https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections).
+
+In order to evaluate the intersection of user-agent support and page support,
+referrers could send RPCs to caches, containing (client information, referent
+URL) tuples. They shouldn't. Doing so widens the sphere of knowledge of this
+information beyond the referrer. This specification does not change the risk:
+
+ 1. It is already possible, and:
+ 2. Referrers have no increased incentive to do so, as the alternative approach
+    (outlined in [Limitations](#limitations)) is both easier to implement and a
+    better user experience, and does not change which entities know which
+    information about which users.
+
+    It is easier to implement because referrers and caches can rely on existing
+    browser support for evaluating media queries, rather than developing
+    client-side code for collecting device information, protocols for
+    serializing this information, and libraries for evaluating it server-side.
+    It is also easier to maintain, as it is more resilient to changes in
+    browser behavior that reduce scripting access to this information.
+
+    It is a better user experience because it can be evaluated in
+    microseconds, rather than hundreds of milliseconds (requiring several RPC
+    hops to the referrer and from there to the cache).
+
+Referent pages could determine whether they've been loaded from a cross-origin
+cache or from origin, and evaluate the media query locally, and compare the
+responses. If they differ, pages might learn some information about user
+preferences on the referrer. For instance, perhaps a User Agent offers
+per-origin configuration of `orientation` or `prefers-color-scheme` and the
+user has configured that specifically for the referrer. For a given user this
+is a fractional bit, but in aggregate may reveal a distribution. Referrers
+should be aware of this risk.
+
+## Alternatives considered
+
+### Syntax
+
+#### UA sniffing
+
+Referrers or caches would need to run arbitrary sniffing algorithms. These can
+be arbitrarily complex (read: Turing complete), hence sandboxing would be
+necessary.
+
+Publishers would need to upload their sniffing algorithms to caches, either
+through bilateral coordination, or by publishing them on the web somewhere
+(e.g. a `.well-known` URI). In some cases, this may be infeasible, e.g. if the
+publisher relies on a closed-source third-party sniffer.
+
+While it would allow for more flexibility in responsiveness than media queries,
+its complexity is in disproportion to the need.
+
+These considerations don't change with regard to the `Sec-CH-UA` proposal.
+
+#### Arbitrary media queries
+
+Referrers could allow any media queries rather than the subset specified above.
+This creates a few risks:
+
+  - Queries might evaluate incorrectly. Of the set of browsers for which a
```

Comment:
This could be worked around using `or` since the parts that browsers don't understand will (I think) evaluate false.

Reply:
I think this means it would fail closed. I added a sentence to clarify this. Please double-check my logic.